include/Enemy.h

```diff
@@ -28,7 +28,8 @@ enum EnemyType {
     etTimidGhost,
     etGremlin,
     etDemon,
-    etNumEnemyTypes
+    etNumEnemyTypes,
+    etWizard
```

Comment:
Please put `etWizard` before `etNumEnemyTypes`, `etNumEnemyTypes` must be last with the way the game's enemy selector works :)